### DISCUSSION TITLE
It is difficult to find the list of event types in MDN

### DISCUSSION THREAD

**Discussion by David263** [mdn/mdn-community#717] [Content]
I typically use the "site:" keyword in browser searches to find information on MDN. I've noticed several times that the information for web page event types (such as "load") is scattered all about. I have yet to find a complete list of event types grouped by functionality. My current specific search is for the events that are triggered when characters are typed into a textarea element. Is there any way to make this often-needed information easier to find?

**Comment by Inventoris** [comment]
Hi @David263! Thank you for your question. First of all, I would recommend looking for events of any element in the documentation for this element. For example, if you are interested in textarea, the events inherent to this tag can be found on its page: https://developer.mozilla.org/en-US/docs/Web/API/HTMLTextAreaElement

If you are interested in finding a list of all events, I would recommend referring to the specification: https://html.spec.whatwg.org/multipage/indices.html#events-2

You can also look at the Event reference https://developer.mozilla.org/en-US/docs/Web/Events

I hope I answered your question!

**Comment by David263** [comment]
Not really. Your MDN links don't have a complete list of events. They have categories of events. I haven't been able to find in MDN a single list of all events supported by browser JavaScript and HTML for the latest browsers.

But I thank you for the obvious work you put into your response.

**Reply by Inventoris** [reply]
@David263 I believe that there is no such list on en-US MDN at the moment. However, I found something similar in the ru version here: https://developer.mozilla.org/ru/docs/Web/Events

I think the best solution would be to initiate an issue, most likely one of the contributers will create such a page in the en-US version.

Perhaps this answer to your question will be more correct.

**Comment by Josh-Cena** [comment]
When is the information about "all supported events" meaningful? For example do you really care about the `redraw` event fired on an `XRCubeLayer`? Events only make sense when listed in the context of the target they are fired on. As @Inventoris said, if you want to find all events fired on a textarea, the `HTMLTextAreaElement` page already provides such information. It lists events specific to textarea, and provides a link to events fired on HTMLElement and Element. Eventually you will find what you need. These lists are big, but a list of literally every event in the browser will only be even larger and less usable.
